pkg/products/threescale/reconciler.go

```diff
@@ -1444,3 +1457,42 @@ func (r *Reconciler) reconcileSubscription(ctx context.Context, serverClient k8s
+		cl.Spec.ApplicationMenu.Section = "Red Hat Applications"
+		cl.Spec.Href = fmt.Sprintf("%v/auth/rhsso/bounce", r.Config.GetHost())
+		cl.Spec.Location = consolev1.ApplicationMenu
+		cl.Spec.Text = "API Management"
```

Comment:
Do you think it would be better to call it "3scale API Management" following https://www.redhat.com/en/products/integration

Reply:
Agree, Changes will come shortly